### ISSUE TITLE
Warning: unmet peer dependency "lodash.deburr@^4.1.0"

### ISSUE THREAD

**Issue by kemsky** [stephanrauh/ngx-extended-pdf-viewer#228]
Latest version of ngx-extended-pdf-viewer produces a warning when installing via yarn:

```
 warning " > ngx-extended-pdf-viewer@2.0.0-beta.2" has unmet peer dependency "lodash.deburr@^4.1.0"
```

**Comment by stephanrauh** [comment]
Oh, you're probably right. I haven't installed yarn, so I can't say for sure. It doesn't happen with npm. However, when I inspected the `package.json`, I spotted a forgotten and obsolete peer dependency.

I've just published version 2.0.0. Please check if it fixes the bug and tell me. 

Thanks in advance,
Stephan

**Comment by kemsky** [comment]
@stephanrauh, thanks!
